server/data/data-library/src/main/java/org/apache/james/user/lib/AbstractJamesUsersRepository.java

```diff
@@ -288,4 +291,9 @@ public void addGroupMapping(MappingSource source, String address) throws Recipie
     public void removeGroupMapping(MappingSource source, String address) throws RecipientRewriteTableException {
         throw new RecipientRewriteTableException("Read-Only RecipientRewriteTable");
     }
+
+    @Override
+    public List<MappingSource> listSources(Mapping mapping) throws RecipientRewriteTableException {
+        return ImmutableList.of();
```

Comment:
By having `listSources` implemented as a default method we would avoid repeating ourselves.